packages/context/src/reflect.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/authenticate
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {inject, Constructor} from '..';
+import 'reflect-metadata';
+
+/*
+ * wrapper to handle reflect api
+ */
+export class ReflectWrapper {
```

Comment:
I find this name not descriptive enough, it's not clear what is being wrapped.

I am proposing something like `NamespacedReflect` or `ScopedReflect`.